tournaments/index.js

```diff
@@ -794,6 +794,7 @@ var commands = {
 			if (params.length < 1) {
 				return this.sendReply("Usage: " + cmd + " <minutes|off>");
 			}
+			if (params[0] === 'Infinity' || params[0] === '0') params[0] = 'off';
```

Comment:
i.e. `isNaN(params[0])` evaluates to `true` if `params[0]` is `'off'`.

You have special-case handling for `off`, and it's clear that it _would_ affect what the command enters in the modlog.